watertap/examples/flowsheets/case_studies/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -0,0 +1,627 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+"""
+Based on flowsheet from:
+
+Flores-Alsina X., Gernaey K.V. and Jeppsson, U. "Benchmarking biological
+nutrient removal in wastewater treatment plants: influence of mathematical model
+assumptions", 2012, Wat. Sci. Tech., Vol. 65 No. 8, pp. 1496-1505
+"""
```

Comment:
Is this the correct reference?

Reply:
No, it's copied from ASM2d. I've deleted it and add another introduction.